test/testapps/BasicTestApp/KeyPressEventComponent.cshtml

```diff
@@ -13,6 +13,7 @@ Type here: <input onkeypress=@OnKeyPressed />
 
     void OnKeyPressed(UIKeyboardEventArgs eventArgs)
     {
+        Console.WriteLine(JsonUtil.Serialize(eventArgs));
```

Comment:
Assuming this just here for crude debugging so ignoring for now.

Reply:
Yeah, I left it there and on touch test so we can watch the full events while testing without we break the current E2E.